frontend/resources/js/components/QuickView.tsx

```diff
@@ -76,7 +73,7 @@ const StyledProductDiv = styled.div`
   }
 `;
 
-const QuickView = ({ product, onClose, siteUrl }: Props) => {
+const QuickView = ({ product, onClose, siteUrl, currencyInfo }: Props) => {
```

Comment:
I wasn't expecting the currency change, just a bugfix that you noticed?

Reply:
Yea, the quick view includes the Price component which was missing the currencyInfo being passed. Weirdly vite isn't throwing typescript errors during compile so I didn't catch it.